<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgTemplate</class>
 <widget class="QDialog" name="dlgTemplate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>424</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Template</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Template Name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="txtName"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Description:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="txtDescription"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Hour(s):</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QListWidget" name="lstHours">
       <property name="selectionMode">
        <enum>QAbstractItemView::MultiSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="cbMon">
       <property name="text">
        <string>Mon</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbTue">
       <property name="text">
        <string>Tue</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbWed">
       <property name="text">
        <string>Wed</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbThu">
       <property name="text">
        <string>Thu</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbFri">
       <property name="text">
        <string>Fri</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbSat">
       <property name="text">
        <string>Sat</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbSun">
       <property name="text">
        <string>Sun</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnOk">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
